pool/web/view/component/component_filter.ml

```diff
@@ -235,7 +306,7 @@ let value_input language input_type ?value () =
              | Lst lst ->
                CCList.filter_map
                  (fun value ->
-                   match value with
+                   match[@warning "-4"] value with
```

Comment:
How many options to add to get rid of the warning suppression?

Reply:
i think 8 times 5